src/plugins/camera/camera_definition_files/generated/camera_definition_files.cpp

```diff
@@ -0,0 +1,13786 @@
+/* Automatic generated file. DO NOT EDIT.
+   The file was generated by cpp_rsc program
+   https://github.com/orex/cpp_rsc
+   This is a source file*/
+
+#include "camera_definition_files.h"
+
+  const char cgoetxml[] = 
+  {
+  '\x3c', '\x3f', '\x78', '\x6d', '\x6c', '\x20', '\x76', '\x65', '\x72', 
+  '\x73', '\x69', '\x6f', '\x6e', '\x3d', '\x22', '\x31', '\x2e', '\x30', 
+  '\x22', '\x20', '\x65', '\x6e', '\x63', '\x6f', '\x64', '\x69', '\x6e', 
+  '\x67', '\x3d', '\x22', '\x55', '\x54', '\x46', '\x2d', '\x38', '\x22', 
+  '\x20', '\x3f', '\x3e', '\x0a', '\x3c', '\x6d', '\x61', '\x76', '\x6c', 
+  '\x69', '\x6e', '\x6b', '\x63', '\x61', '\x6d', '\x65', '\x72', '\x61', 
+  '\x3e', '\x0a', '\x20', '\x20', '\x20', '\x20', '\x3c', '\x64', '\x65', 
+  '\x66', '\x69', '\x6e', '\x69', '\x74', '\x69', '\x6f', '\x6e', '\x20', 
+  '\x76', '\x65', '\x72', '\x73', '\x69', '\x6f', '\x6e', '\x3d', '\x22', 
+  '\x36', '\x22', '\x3e', '\x0a', '\x20', '\x20', '\x20', '\x20', '\x20', 
+  '\x20', '\x20', '\x20', '\x3c', '\x6d', '\x6f', '\x64', '\x65', '\x6c', 
+  '\x3e', '\x43', '\x47', '\x4f', '\x45', '\x54', '\x3c', '\x2f', '\x6d', 
```

Comment:
```suggestion
  '\x3e', '\x42', '\x47', '\x4f', '\x45', '\x54', '\x3c', '\x2f', '\x6d', 
```
Pretty sure! :smile: